pgo-scheduler/scheduler/policy.go

```diff
@@ -191,3 +191,12 @@ func (p PolicyJob) Run() {
 		return
 	}
 }
+
+// getCCPImagePrefix checks whether the CCPImagePrefix value is set in the pgcluster CRD.
+// If so, this value is returned to be used, otherwise the value from the job is used.
+func getCCPImagePrefix(cluster crv1.Pgcluster, jobPrefix string) string {
+	if cluster.Spec.CCPImagePrefix != "" {
+		return cluster.Spec.CCPImagePrefix
+	}
+	return jobPrefix
+}
```

Comment:
Isn't this effectively the same logic as the `GetImagePrefix`function above? I feel like this could be DRY'ed up a little.

Reply:
absolutely, didn't catch this during my rewrite of `GetImagePrefix` on Friday. It's corrected now, thanks.